### ISSUE TITLE
Prepare script triggers when installing this lib as a dependency

### ISSUE THREAD

**Issue by jimblue** [ismay/stylelint-no-unsupported-browser-features#151]
Hello,

I just tried the latest release 5.0.0 and I'm getting the following error :

```
npm ERR! code 127
npm ERR! path /Users/jim/Dev/websites/immersion-pictures/node_modules/stylelint-no-unsupported-browser-features
npm ERR! command failed
npm ERR! command sh -c husky install
npm ERR! sh: husky: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jimblue/.cache/npm/_logs/2021-05-22T15_01_44_919Z-debug.log
```

**Comment by ismay** [comment]
Hmm that's weird, I wonder if the `prepare` step is also triggered when you install this module as a dependency. That's not supposed to happen. I'm using the recommended `prepare` script, from the husky docs: https://typicode.github.io/husky/#/?id=install.

Which command led to this error?

**Comment by ismay** [comment]
All right, to be on the safe side I've just pushed a quick fix that gets rid of the prepare script (5.0.1). However, to properly debug this I would like to know what led to this error. Could you let me know which command lead to the error you encountered @jimblue & @ilyub?

Because it _seems_ like the prepare script shouldn't be triggered if you're installing this module as a dependency, but it's a little unclear. Related:

- https://github.com/npm/cli/issues/2152
- https://docs.npmjs.com/cli/v6/using-npm/scripts#prepare-and-prepublish
- https://github.com/esm-bundle/react-dom/issues/152
- https://github.com/typicode/husky/issues/914

**Comment by ismay** [comment]
I've verified that 5.0.1 fixes this issue. I'll leave the issue open as a reminder to myself to fix the automation of the husky installs for development purposes. However for users of this lib the bug should be fixed now.

**Comment by ismay** [comment]
False alarm. 5.0.0 included a postinstall instead of a prepare script. So this can be considered fixed.
